OpenFold container

tonykew opened the pull request:
Use numpy 1.x
Add libaio
x86_64 and ARM64 build working
Ran Inference tests on x86_64 and ARM64
Ran Training on 8 GPU DGX (x86_64) node:
  Completed epoch 0 training
  Restarted training from epoch 0 checkpoint AOK
Slurm examples for x86_64 and ARM64

NOTE: The following files have "raw" github URLs that will have to be fixed for production:

BUILD-ARM64.md
README.md

Tony

tonykew committed:
OpenFold Container example

Use numpy 1.x
Add libaio
x86_64 and ARM64 build working
Unfortunately here are several deprecation and future warnings with
the ARM64 build that cannot be easily supresssed - needs code changes
Prevent attempting to run X86 binaries on ARM64
Create an empty tuning dir for triton
Added a fix for a "df" error when running OpenFold
Ran Inference tests on x86_64 and ARM64
Ran Training on 8 GPU DGX (x86_64) node:
  Completed ephoch 0 training
  Restarted training from epoch 0 checkpoint AOK
Slurm examples for x86_64 and ARM64

NOTE: The following files have "raw" github URLs that will have to be fixed
for production:

BUILD-ARM64.md
README.md

Tony

BlazeCheesegrater commented on containers/2_ApplicationSpecific/OpenFold/EXAMPLES.md:
````diff
@@ -0,0 +1,963 @@
+mkdir -p ./output
+```
+
+Start the container, with the "./output" directory a the top level output
````
```suggestion
Start the container, with the "./output" directory as the top level output
```

tonykew committed:
SLURM_NPROCS fix

SLURM_NPROCS is not necessarily a defined environment variable when a
node is salloced - this change deals with that case

Tony

tonykew committed:
Fix typos

Tony

tonykew committed:
Merge branch 'main' into OpenFold

tonykew committed:
Merge branch 'main' into OpenFold

tonykew committed:
Merge branch 'main' into OpenFold

tonykew committed:
Updated bind mounts

Tony
